app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalListItem.vue

```diff
@@ -0,0 +1,415 @@
+<template>
+  <div>
+    <div v-for="portal in portals" :key="portal.id" class="portal">
```

Comment:
Create the Portal Item as a separate component.

Reply:
It's already in a separate component.

Reply:
@iamsivin Can't we iterate `portals` array in `AllPortal` component itself?

Reply:
In this PR we are not linking it with routes thats why i don't add it. Will add it.

Reply:
@muhsin-k I think we use it in `app/javascript/dashboard/routes/dashboard/helpcenter/pages/portals/ListAllPortals.vue` this file or here it self.